src/RemoteLRS.php

```diff
@@ -265,40 +257,48 @@ private function _parseMultipart($boundary, $content) {
     // as suggested here: http://php.net/manual/en/function.http-parse-headers.php#112986
     //
     // adapted to private method, and force headers to lowercase for easy detection
+    // 
+    // Now reworked so that the variable names make a bit more sense.
     //
     private function _parseHeaders($raw_headers) {
         $headers = array();
         $key = ''; // [+]
 
-        foreach(explode("\n", $raw_headers) as $i => $h) {
-            $h = explode(':', $h, 2);
-            $h[0] = strtolower($h[0]);
+        $raw_headers = explode("\n", $raw_headers);
+        $raw_headers = array_filter($raw_headers);
 
-            if (isset($h[1])) {
-                if (! isset($headers[$h[0]])) {
-                    $headers[$h[0]] = trim($h[1]);
-                }
-                elseif (is_array($headers[$h[0]])) {
-                    // $tmp = array_merge($headers[$h[0]], array(trim($h[1]))); // [-]
-                    // $headers[$h[0]] = $tmp; // [-]
-                    $headers[$h[0]] = array_merge($headers[$h[0]], array(trim($h[1]))); // [+]
+        foreach($raw_headers as $i => $header) {
+
+            $header = explode(':', $header, 2);
+ 
+            $name = strtolower($header[0]);
+            $value = (isset($header[1])) ? $header[1] : false;
+
+            if($value === false) {
+
+                if (substr($name, 0, 1) == "\t") {
```

Comment:
This is a good example of style inconsistency, 3 lines apart in the change set, in one that doesn't seem necessary at all.